lib/js/components/RichList/RichListItem/RichListItem.vue

```diff
@@ -0,0 +1,424 @@
+	display: flex;
+	flex: 1;
+	flex-direction: column;
+	justify-content: center;
+	max-width: 100%;
+	min-height: 62px;
+	padding: $space-xxxxs $space-xxxxs $space-xxxxs 0;
+	pointer-events: none;
+	position: relative;
+
+	@media #{breakpoint-s()} {
+		padding: 0 $space-xxxxs;
+	}
+
+	&.-interactive {
+		cursor: pointer;
+		pointer-events: initial;
+	}
+
+	&.-dimmed {
+		opacity: 0.5;
+	}
+
+	&.-flat {
+		background: $color-neutral-background-ghost;
+
+		&:hover {
+			background: $color-neutral-background-ghost-hovered;
+		}
+
+		#{$root}__wrapper {
+			border: none;
+		}
+	}
+
+	&:not(.-flat) {
+		@each $color, $value in $rich-list-item-border-colors {
+			&.-border-#{$color} {
+				#{$root}__border {
+					background-color: $value;
+				}
+			}
+		}
+
+		border: 1px solid $color-neutral-border-weak;
+		border-radius: $radius-s;
+
+		&:hover {
+			background: $color-neutral-background-hovered;
+			opacity: 1;
+		}
+
+		#{$root}__border {
+			border-radius: $radius-s 0 0 $radius-s;
+			height: calc(100% + 2px);
+			left: -1px;
+			position: absolute;
+			top: -1px;
+			width: $space-xxxxs;
+		}
+	}
+
+	&__container {
+		align-items: flex-start;
+		display: flex;
+		width: 100%;
+
+		@media #{breakpoint-s()} {
+			align-items: center;
+		}
+	}
+
+	&__dragAndDrop,
+	&__iconWrapper {
+		align-items: center;
+		display: flex;
+		padding: $space-xs $space-xxxs 0 $space-xs;
+
+		@media #{breakpoint-s()} {
+			padding: $space-xxxs $space-xxxs $space-xxxs $space-s;
+		}
+	}
+
+	&__content {
+		flex: 2 0 0;
+		min-width: 0;
+		padding: 0 $space-xxs 0 $space-xxxs;
+
+		@media #{breakpoint-s()} {
+			padding: 0 $space-xxs;
+		}
+	}
+
+	&__metaData {
+		align-items: center;
+		display: flex;
+		flex: 1;
+		padding-left: $space-xs;
+
+		@media #{breakpoint-s()} {
+			justify-content: flex-start;
+			padding: 0;
+		}
+
+		&.-hideOnMobile {
+			display: none;
+
+			@media #{breakpoint-s()} {
+				display: flex;
+			}
+		}
+
+		&.-visibleOnMobile {
+			display: flex;
+
+			@media #{breakpoint-s()} {
+				display: none;
+			}
+		}
+	}
+
+	&__trailingSlot {
+		align-items: center;
+		display: flex;
+		padding: 0;
+
+		@media #{breakpoint-s()} {
+			padding-left: $space-xxs;
+		}
+	}
+
+	&__icon {
+		@each $color, $value in $rich-list-item-icon-colors {
+			&.-icon-color-#{$color} {
+				color: $value;
+			}
+		}
+
+		color: $color-neutral-icon-weak;
+	}
+
+	&__dragAndDropIcon {
+		color: $color-neutral-icon-weak;
+
+		&:hover {
+			color: $color-neutral-icon-weak-hovered;
+			cursor: grab;
+		}
+
+		&:active {
+			cursor: grabbing;
+		}
+	}
+
+	&.-loading {
+		cursor: initial;
+		opacity: 0.5;
+		pointer-events: none;
+	}
+
+	&.-small {
+		min-height: 44px;
+		padding: $space-xxxxs $space-xxxs $space-xxxxs $space-xxxxs;
+
+		@media #{breakpoint-s()} {
+			padding: 0 $space-xxxs 0 0;
+		}
+
+		#{$root}__dragAndDrop,
+		#{$root}__iconWrapper {
+			padding: 0 0 0 $space-xs;
+
+			@media #{breakpoint-s()} {
+				padding: $space-xxxs 0 $space-xxxs $space-xs;
```

Comment:
This is an inconsistency how you implemented those paddings. For mobile view, you moved them to root element, but for desktop you set them on icons.

Reply:
what? `but for desktop you set them on icons.` no, on wrapper in both cases

Reply:
Ok. You are right.